### ISSUE TITLE
Parse and use platform.txt

### ISSUE THREAD

**Issue by scottdarch** [scottdarch/Arturo#11]
INO manually assembled make recipes. Arduino now supplied ready made recipe templates in the platforms.txt file. Parse this and then replace most of the makefile logic in build.py to use these templates.

**Comment by photodude** [comment]
:1:
Sounds like a great idea for Arturo 2 refactoring

**Comment by scottdarch** [comment]
Yep. If you look at the develop branch you'll see I've already got this built. I'm currently working through library builds which is turning into a real pain.
